src/persist-types/src/stats.rs

```diff
@@ -61,6 +62,25 @@ pub trait DynStats: std::fmt::Debug + Send + Sync + 'static {
     fn into_proto(&self) -> ProtoDynStats;
 }
 
+mod private {
+    /// Statistics serialization costs
+    pub trait StatsCost {
+        /// A proxy for the cost to serialize these stats, in bytes.
+        ///
+        /// WIP should we tie this to some specific meaning: e.g. the size of
```

Comment:
@bkirwi any thoughts on this?

Reply:
Agree with everything you wrote in the comment, basically. I'm fine with defining this pretty loosely for now, and after the next round of testing we should have a better idea of how this behaves in real life / whether it's worth investing the time to make this more strict / predictable.

Reply:
👍 swapped it to TODO(mfp) so we don't forget to come back to it